From bb60ab47e1bcf02c1e54498aaf99cccd7811fb3e Mon Sep 17 00:00:00 2001
From: "zongren.yin" <zongren.yin@amlogic.com>
Date: Wed, 22 Nov 2023 09:03:25 +0800
Subject: [PATCH 1/5] uboot: No signal while tvtest display via vbyone [1/1]

PD#SWPL-146941

Problem:
No signal while tvtest display via vbyone

Solution:
Set panel1_type as vbyone_2

Fixed conflicts:
NA

Verify:
local t7c build

Signed-off-by: zongren.yin <zongren.yin@amlogic.com>
Change-Id: I7d76629b4f02318c0a12155cb4f6f952bcac0ac3
---
 board/amlogic/configs/t7_an400_lpddr4x_debian.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/amlogic/configs/t7_an400_lpddr4x_debian.h b/board/amlogic/configs/t7_an400_lpddr4x_debian.h
index f68be8039ee..7bced9258d4 100644
--- a/board/amlogic/configs/t7_an400_lpddr4x_debian.h
+++ b/board/amlogic/configs/t7_an400_lpddr4x_debian.h
@@ -107,7 +107,7 @@
 #else
 #define CONFIG_EXTRA_HDMI_ENV_SETTINGS \
 	"panel_type=vbyone_0\0" \
-	"panel1_type=vbyone_1\0" \
+	"panel1_type=vbyone_2\0" \
 	"panel2_type=lvds_1\0" \
 	"lcd1_ctrl=0x00000000\0" \
 	"lcd2_ctrl=0x00000000\0" \
-- 
2.25.1

